examples/wave/wave-op-mpi.py

```diff
@@ -91,14 +94,29 @@ def wave_flux(dcoll, c, w_tpair):
         v=0.5 * v_jump * normal,
     )
 
-    return op.project(dcoll, w_tpair.dd, "all_faces", c*flux_weak)
+    return op.project(dcoll, dd, dd.with_dtag("all_faces"), c*flux_weak)
 
 
-def wave_operator(dcoll, c, w):
-    u = w.u
-    v = w.v
+def wave_operator(dcoll, c, w, quad_tag=None):
```

Comment:
How hard would it be to also test with a non-affine mesh?

Reply:
IMO this is out of scope and I'd rather not mess too much further with the existing examples.

Reply:
Nvm, I think I came up with something nonaffine for the wave op example :slightly_smiling_face: 
![image](https://user-images.githubusercontent.com/14180421/149248708-23b4d39b-d8e5-4f5b-baff-513d3591a94c.png)

Reply:
https://github.com/inducer/grudge/pull/210/commits/d468071d582918d3f3c7b6962bd0168ac5560fb7